src/SFA.DAS.Tools.Servicebus.Support.Application/Queue/Queries/GetMessageCountPerUser/GetMessageCountPerUserQueryHandler.cs

```diff
@@ -0,0 +1,41 @@
+﻿using SFA.DAS.Tools.Servicebus.Support.Domain;
+using SFA.DAS.Tools.Servicebus.Support.Infrastructure.Services.CosmosDb;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Tools.Servicebus.Support.Application.Queue.Queries.GetMessageCountPerUser
+{
+    public class GetMessageCountPerUserQueryHandler : IQueryHandler<GetMessageCountPerUserQuery, GetMessageCountPerUserQueryResponse>
+    {
+        private readonly ICosmosMessageDbContext _cosmosDbContext;
+
+        public GetMessageCountPerUserQueryHandler(ICosmosMessageDbContext cosmosDbContext)
+        {
+            _cosmosDbContext = cosmosDbContext;
+        }
+
+        public async Task<GetMessageCountPerUserQueryResponse> Handle(GetMessageCountPerUserQuery query)
+        {
+            var messageCountPerUser = await _cosmosDbContext.GetMessageCountPerUser();
```

Comment:
Probably should have async on the end for consistency?
```suggestion
            var messageCountPerUser = await _cosmosDbContext.GetMessageCountPerUserAsync();
```